### DISCUSSION TITLE
Translating FormType choice options

### DISCUSSION THREAD

**Discussion by anzweidrej** [symfony/symfony#53750] [Q&A] [answered]
My question is what is the correct way to make the form 'choices' options translatable, so that the strings can be extracted via the "translation:extract" command.

Thank you in advance.

**Comment by MatTheCat** [comment, marked as answer]
You should be able to leverage `TranslatableMessage` as shown here: https://github.com/symfony/symfony/issues/50744#issuecomment-1602706589

**Reply by anzweidrej** [reply]
```php
use Symfony\Component\Translation\TranslatableMessage;
use Symfony\Component\Form\Extension\Core\Type\ChoiceType;

$builder
->add('foo', ChoiceType::class, [
   'choices' => ['option1' => new TranslatableMessage('bar', [], 'messages')]
```
^^ does work, thx man.
